js/functions.js

```diff
@@ -0,0 +1,56 @@
+function checkCommentLength(comment, length) {
+  if (comment.length <= length) {
+    return true;
+  }
+  return false;
+}
+
+checkCommentLength('проверяемая строка', 20);
+checkCommentLength('проверяемая строка', 18);
+checkCommentLength('проверяемая строка', 10);
+
+function checkPalindromm(palindrom) {
+  palindrom = palindrom.toLowerCase().replaceAll(' ', '');
```

Comment:
Идет переприсвоение параметра, это в целом не страшно, но был бы линтер пожестче, он бы поругал
```suggestion
  const formatted = palindrom.toLowerCase().replaceAll(' ', '');
```